packages/monaco-editor-wrapper/src/editorAppBase.ts

```diff
@@ -201,14 +207,16 @@ export abstract class EditorAppBase {
         }
     }
 
-    updateMonacoEditorOptions(options: editor.IEditorOptions & editor.IGlobalEditorOptions) {
-        this.editor?.updateOptions(options);
+    async updateUserConfiguration(config: UserConfiguration) {
```

Comment:
If this is internal perhaps another name would be better, this is confusing that there are two functions with the same name. Makes it a bit weird to find there's both an import and a definition in the same context.

Reply:
I will rename the import to `vscodeUpdateUserConfiguration`